docs/rules/valid-jsdoc.md

```diff
@@ -188,6 +188,85 @@ By default ESLint requires you to specify `type` for `@return` tag for every doc
 }]
 ```
 
+#### preferType
+
+It will validate all the types from jsdoc with the options setup by the user. Inside the options, key should be what the type you want to check and the value of it should be what the expected type should be. Please make all the keys in lowercase as we compare them by making the type from jsdoc to lowercase and the value should be case sensitive as in how you expect it to be.
```

Comment:
But what if inside the code somebody writes `strinG` by mistake then we will not be able to catch it. That why I always want to compare the lowercased value from the jsdoc with the key here.